tests/Squidex.Domain.Apps.Core.Tests/Model/Contents/WorkflowJsonTests.cs

```diff
@@ -19,8 +24,33 @@ public void Should_serialize_and_deserialize()
             var workflow = Workflow.Default;
 
             var serialized = workflow.SerializeAndDeserialize();
+            var test = workflow.GetTransitions(Status.Archived);
```

Comment:
Whats that used for?

Reply:
Sorry, that was a typo from my side. Apologies.